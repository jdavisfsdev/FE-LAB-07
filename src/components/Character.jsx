import React from 'react';

export default function Character({ name }) {
  return (
    <div>
      <p>{name}</p>   
    </div>
  );
}
