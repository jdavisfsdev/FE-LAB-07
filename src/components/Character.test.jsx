import React from 'react';
import { render } from '@testing-library/react';
import Character from './Character';

describe('Character component snapshot', () => {
  it('render character', () => {
    const { asFragment } = render(
      <Character />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
