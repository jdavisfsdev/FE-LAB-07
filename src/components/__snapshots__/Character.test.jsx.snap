// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component snapshot render character 1`] = `
<DocumentFragment>
  <div>
    <p />
  </div>
</DocumentFragment>
`;
